docs/site/Deployment-guide-Cloud-Foundy.md

```diff
@@ -0,0 +1,244 @@
+---
+lang: en
+title: 'How to deploy LoopBack 4 app to Cloud Foundry'
```

Comment:
If putting this as the root layer, the title should be consistent with the rest of the titles.  
Suggestion: Deploying to IBM Cloud (or something along that line)

After the title has been finalized, might want to change the file name as well.  (typo in the file name as well)